Patches/Not Only Just A Cannon/Patch_Cannon_Turret.xml

```diff
@@ -0,0 +1,850 @@
+          </FireModes>
+        </li>
+
+        <li Class="PatchOperationAdd">
+          <xpath>/Defs/ThingDef[defName="Forted_Cannon"]</xpath>
+          <value>
+            <thingClass>CombatExtended.Building_TurretGunCE</thingClass>
+          </value>
+        </li>
+		
+		<li Class="PatchOperationAdd">
+          <xpath>/Defs/ThingDef[defName="Forted_Cannon"]</xpath>
+          <value>
+            <specialDisplayRadius>86</specialDisplayRadius>
+          </value>
+        </li>
+
+        <li Class="PatchOperationRemove">
+          <xpath>/Defs/ThingDef[defName="Weapon_FortedCannon"]/comps/li[@Class="CompProperties_ChangeableProjectile"]</xpath>
+        </li>
+		
+        <li Class="PatchOperationRemove">
+          <xpath>/Defs/ThingDef[defName="Weapon_FortedCannon"]/building</xpath>
+        </li>
+		
+		
+        <li Class="PatchOperationAdd">
+          <xpath>/Defs/ThingDef[defName="Forted_Cannon"]</xpath>
+          <value>
+            <fillPercent>0.85</fillPercent>
+          </value>
+        </li>
+		
+		
+        <li Class="PatchOperationReplace">
+          <xpath>/Defs/ThingDef[defName="Forted_Cannon"]/building/turretBurstCooldownTime</xpath>
+          <value>
+            <turretBurstCooldownTime>1</turretBurstCooldownTime>
+          </value>
+        </li>
+		
+		
+        <li Class="PatchOperationAdd">
+          <xpath>/Defs/ThingDef[defName="Forted_Cannon"]/statBases</xpath>
+          <value>
+            <ShootingAccuracyTurret>1</ShootingAccuracyTurret>
+          </value>
+        </li>
+		
+		<!--   COEHORN   -->
+		
+		<li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+          <defName>Weapon_Coehorn</defName>
+
+          <statBases>
+			  <SightsEfficiency>0.5</SightsEfficiency>
+			  <ShotSpread>0.3</ShotSpread>
+			  <SwayFactor>1.34</SwayFactor>
+			  <RangedWeapon_Cooldown>2.71</RangedWeapon_Cooldown>
+          </statBases>
+			
+          <Properties>
+			<verbClass>CombatExtended.Verb_ShootMortarCE</verbClass>
+			<forceNormalTimeSpeed>false</forceNormalTimeSpeed>
+			<hasStandardCommand>true</hasStandardCommand>
+			<defaultProjectile>Bullet_164mmCannonShell_HE_Mortar</defaultProjectile>
+			<warmupTime>3</warmupTime>
+			<minRange>15</minRange>
+			<range>50</range>
+			<burstShotCount>1</burstShotCount>
+			<soundCast>MortarBomb_Explode</soundCast>
+			<muzzleFlashScale>32</muzzleFlashScale>
+			<circularError>1.5</circularError>
+			<indirectFirePenalty>0.6</indirectFirePenalty>
+			<targetParams>
+			  <canTargetLocations>true</canTargetLocations>
+			</targetParams>
+          </Properties>
+			
+          <AmmoUser>
+            <magazineSize>1</magazineSize>
+            <reloadTime>10</reloadTime>
+            <ammoSet>AmmoSet_164mmCannonBall_MortarSmall</ammoSet>
+          </AmmoUser>
+			
+          <FireModes>
+			<aiUseBurstMode>FALSE</aiUseBurstMode>
+			<aiAimMode>AimedShot</aiAimMode>
+			<noSnapshot>true</noSnapshot>
+			<noSingleShot>true</noSingleShot>
```

Comment:
These guns should match the existing flak cannon in that they should only have single shot mode appear on the gizmo, as opposed to fully automatic like they do now.